src/main/java/org/terasology/potions/events/BeforeDrinkPotionEvent.java

```diff
@@ -173,11 +294,20 @@ public double getDurationResultValue() {
         return result;
     }
 
+    /**
+     * Informs whether or not this event has been consumed.
+     *
+     * @return True if the the event has been consumed, false otherwise.
+     */
     @Override
     public boolean isConsumed() {
```

Comment:
you may checkout `{@inheritDoc}` and use cases. this case seems like the javadoc should be same in the parent class (the consumable event). If you don't write javadoc at all, it is automatically copied from the override method.